sql/catalyst/src/main/scala/org/apache/spark/sql/catalyst/expressions/objects/objects.scala

```diff
@@ -555,8 +555,27 @@ case class NewInstance(
   }
 
   override def eval(input: InternalRow): Any = {
```

Comment:
I believe this is now matched with codegen part.

Reply:
I borrowed some codes from `InvokeLike`. It's a bit complicated to share the same codebase.